Presentation.Web/app/components/it-system/it-interface/it-interface-edit-tab-main.controller.ts

```diff
@@ -60,6 +130,106 @@
                         }
                     };
                 }
+
+                function selectLazyLoadingForSystem(url, paramAry) {
+                    return {
+                        allowClear: true,
```

Comment:
Denne her store onde blok var også andre steder. Måske koden skulle trækkes ud i en lazy loading service... ikke sandt @JacobMalling ?

Reply:
"select" delen står åbenbart for select2 valgboksene. Så det er nu trukket ud i en service der så kan genbruges alle steder der bruges select2